src/procedures/move-data-disk.html

```diff
@@ -0,0 +1,37 @@
+---
+tags:
+  - add-sdd-existing-installation
+order: 10
+procedures:
+  - title: Configuring Home Assistant and moving your data
+    steps:
+      - title: Go to the storage configuration page
+        image: ha-config-storage.png
+        content: |
+          <ul>
+            <li>
+              Open your Home Assistant instance and navigate to the storage configuration page using the button below:<br><br>
+              <a href="https://my.home-assistant.io/redirect/storage/" target="_blank"><img
+                src="https://my.home-assistant.io/badges/storage.svg"
+                alt="Open your Home Assistant instance and show storage information."
+                style="width: initial"
+              /></a>
+              <br><br>
+              Alternative, you can go to the storage page via Configuration, System, Storage.
+            </li>
+          </ul>
+      - title: Open the Move to Datadisk dialog
+        image: ha-config-storage-overflow-menu.png
+        content: |
+          <ul>
+            <li>Open the overflow menu by clicking on X in the top-right of the screen.</li>
```

Comment:
Can we use the UTF-8 symbol ⋮ safely for the three dot menu thing? It seems to work here.,,

```suggestion
            <li>Open the overflow menu by clicking on ⋮ in the top-right of the screen.</li>
```

It seems we use "three dots" in the official documentation.

@frenck thoughts?

Reply:
oh that was a placeholder. I can just get the SVG

Reply:
I added the SVG icon so it's very clear now.